baseband/mark5b/base.py

```diff
@@ -319,10 +319,9 @@ def _make_frame(self, index):
         # Set up header for new frame.
         header = self.header0.copy()
         # Update time and frame_nr in one go.
-        # (Note: could also pass on frame rate instead of explicit frame)
-        header.set_time(time=self.start_time + index / self._framerate * u.s,
-                        frame_nr=((self.header0['frame_nr'] + index) %
-                                  self._framerate))
+        framerate = u.Quantity(self._framerate, u.Hz)
```

Comment:
Is this faster or better handles odd cases than `framerate = self._framerate * u.Hz`?

Reply:
it is marginally faster. Probably not worth the lack of clarity, though, so I'll undo.